### PR TITLE
Add section about environment variables needed to the readme

### PR DESCRIPTION
**Pull request by fiennyangeln**
Bug #27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,19 @@ cd bug-issue-ui
 yarn
 ```
 
+### Environment variables
+
+To get started local development, create a file in the root of the repo named
+`.env` with the following content.
+
+```bash
+GITHUB_PERSONAL_ACCESS_TOKEN = <insert personal access token here>
```
Comment (helfi92) [inline README.md]:
```bash
GITHUB_PERSONAL_API_TOKEN=<your_github_api_token>
```

File: README.md [hunk 2]
```diff
@@ -25,6 +25,19 @@ cd bug-issue-ui
...
+```
+
+To get a github personal access token, go to [Github Settings Page](https://github.com/settings/profile),
+click on the Developer Settings, then click on Personal Access Token, and click on Generate New Token.
+For the scopes, select all scopes under `repo` category. After that, click Generate Token and copy the token to the above.
```
Comment (helfi92) [inline README.md]: How about:

Generate a personal token [here](https://github.com/settings/tokens). When prompted about scopes, **access to public repositories** is the only one required.

File: README.md [hunk 3]
```diff
@@ -25,6 +25,19 @@ cd bug-issue-ui
 yarn
 ```
 
+### Environment variables
+
+To get started local development, create a file in the root of the repo named
```
Comment (eliperelman) [inline README.md]: "To get started with local development"

File: README.md [hunk 4]
```diff
@@ -25,6 +25,19 @@ cd bug-issue-ui
...
+`.env` with the following content.
```
Comment (eliperelman) [inline README.md]: Add the sentence:

"You can also set the variables in your shell environment."

File: README.md [hunk 5]
```diff
@@ -25,6 +25,19 @@ cd bug-issue-ui
...
+
+```bash
+GITHUB_PERSONAL_ACCESS_TOKEN = <insert personal access token here>
+```
+
+To get a github personal access token, go to [Github Settings Page](https://github.com/settings/profile),
```
Comment (eliperelman) [inline README.md]: Use the casing "GitHub" when referring to the GitHub name.

File: README.md (hunk 1 above)
Comment (fiennyangeln) [reply README.md]: Ok, i will change the env name in other PR then.

**Pull request merged**